puppet/modules/profiles/manifests/monitoring/client.pp

```diff
@@ -0,0 +1,32 @@
+class profiles::monitoring::client (
+  String[1] $prometheus_username,
+  String[1] $prometheus_password,
+  Stdlib::HTTPUrl $prometheus_url = 'https://prometheus-prod-01-eu-west-0.grafana.net/api/prom/push',
+  Array[Hash] $blackbox_targets = [],
```

Comment:
this can be set to something like:
```
monitoring::blackbox_targets:
  - name: jenkins
    address: https://ci.theforeman.org
    module: http
  - name: jenkins-v6
    address: https://ci.theforeman.org
    module: http6
```